common/src/main/resources/lang/fr.json

```diff
@@ -51,6 +51,7 @@
   "format_start_limit": "Votre hôte a limité le rayon de pré-génération maximum à %s pour éviter une utilisation excessive de l'espace disque. Réduisez le rayon ou contactez votre hôte si vous souhaitez que cette limite soit supprimée.",
   "format_started_already": "La tâche a déjà commencé pour %s !",
   "format_world": "Le monde est passé à %s.",
+  "format_worldborder_too_large": "&eNotice: Your world border is currently set larger than the maximum of %s configured in server.properties. This is normally an error and can cause issues.",
```

Comment:
&eNotice : La bordure de votre monde est actuellement plus grande que le maximum de %s configuré dans server.properties. Il s'agit normalement d'une erreur qui peut entraîner des problèmes.

Reply:
Added manually. Thank you!